packages/rpc-core/src/rpc-methods/__tests__/simulate-transaction-test.ts

```diff
@@ -0,0 +1,717 @@
+import { base58, fixSerializer } from '@metaplex-foundation/umi-serializers';
+import { Base58EncodedAddress } from '@solana/addresses';
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import { SolanaJsonRpcErrorCode } from '@solana/rpc-transport/dist/types/json-rpc-errors';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import { Base64EncodedWireTransaction } from '@solana/transactions';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { Base58EncodedBytes, Commitment } from '../common';
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+function getMockTransactionMessage({
```

Comment:
Related comment: https://github.com/solana-labs/solana-web3.js/pull/1504#discussion_r1300697601

Reply:
Probably the same conclusion here - we need to test with different blockhashes